Sources/NIOFileSystem/FileSystem.swift

```diff
@@ -17,6 +17,7 @@
 import Atomics
 import NIOCore
 @preconcurrency import SystemPackage
+import class Foundation.FileManager
```

Comment:
The `NIOFilsystem` module is intentionally not depending on `Foundation`. Is there a way to solve this without using `Foundation` here?

Reply:
I didn't know that, will revert this.

Reply:
Done